R/mod_reportsTab.R

```diff
@@ -84,12 +85,13 @@ reportsTab <- function(input, output, session, charts, data, mapping){
         mapping = mapping(), 
         charts=charts_keep()
       )
-      
+      show_modal_spinner(text="Creating html report. This might take a while for multiple charts/large data sets.") # show the modal window
       rmarkdown::render(tempReport,
                         output_file = file,
                         params = params,  ## pass in params
                         envir = new.env(parent = globalenv())  ## eval in child of global env
       )
+      remove_modal_spinner() # remove it when done
```

Comment:
This line doesn't run if report fails - Demographics Table, Shift Table - and spinner just spins.

![image](https://user-images.githubusercontent.com/5428548/122450091-9cdfca00-cf74-11eb-98ce-a87bf6b93d47.png)

Reply:
Can we just drop these two modules for this release?  They don't work on my workstation.

Reply:
Yeah, Shift Table is no longer in the default build after https://github.com/SafetyGraphics/safetyCharts/pull/67

Reply:
> This line doesn't run if report fails - Demographics Table, Shift Table - and spinner just spins.

Yeah, I'm not sure the best workflow here. I guess at the very least we should have some trycatch() logic that throws a decent message to the modal if the export fails.

I think I'm going to leave #517 for a different PR that deals with some of the other issues in the export. I'll close this one, and make a new PR that just resolves #576.